package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@unly/serverless-plugin-db-backups",
```

Comment:
`@unly/serverless-plugin-dynamodb-backups`

Note: I need to change the github repository URL to match this change